packages/aws-cdk/lib/api/logs/find-cloudwatch-logs.ts

```diff
@@ -0,0 +1,101 @@
+import * as cxapi from '@aws-cdk/cx-api';
+import { Mode, SdkProvider, ISDK } from '../aws-auth';
+import { prepareSdkWithLookupRoleFor } from '../cloudformation-deployments';
+import { EvaluateCloudFormationTemplate, LazyListStackResources } from '../evaluate-cloudformation-template';
+
+// resource types that have associated CloudWatch Log Groups that should _not_ be monitored
+const IGNORE_LOGS_RESOURCE_TYPES = ['AWS::EC2::FlowLog', 'AWS::CloudTrail::Trail', 'AWS::CodeBuild::Project'];
+
+// Resource types that will create a CloudWatch log group with a specific name if one is not provided.
+// The keys are CFN resource types, and the values are the names of the physical name property of that resource.
```

Comment:
Don't you need to account for the prefix? Or are you guaranteeing that it will **always** be `/aws/<servicename>/` ?

Because I can assure you, AWS don't roll that way 🤣

Reply:
I could update this to include the prefix to more future proof this.

Reply:
Done.